doc/manual/src/architecture/architecture.md

```diff
@@ -0,0 +1,42 @@
+# Architecture
+
+This chapter describes how Nix works.
+It should help users understand why Nix behaves as it does, and it should help developers understand how to modify Nix and how to write similar tools.
+
+## Overview
+
+Nix consists of hierarchical [layers](https://en.m.wikipedia.org/wiki/Multitier_architecture#Layers).
```

Comment:
Links to mobile version of Wikipedia (here and in the following) probably unintentional?

Reply:
Mobile Wikipedia is intentional, as it is less visual noise and page data.